cfgov/v1/wagtail_hooks.py

```diff
@@ -128,46 +129,21 @@ def configure_page_revision(page, is_sharing, is_live):
 
 
 def get_akamai_credentials():
```

Comment:
You're right, this should probably live somewhere else. I'm not sure where to suggest, right off.